contracts/contracts/CAPE.sol

```diff
@@ -213,6 +213,7 @@ contract CAPE is RecordsMerkleTree, RootStore, AssetRegistry, ReentrancyGuard {
     /// @param erc20Address address of the ERC-20 token corresponding to the deposit
     function depositErc20(RecordOpening memory ro, address erc20Address) public nonReentrant {
```

Comment:
Shall we remove the `address erc20Address` in this PR?

Reply:
We prefer not to for now to avoid breaking the ui code. @sveitser will create another ticket to do this later.

Reply:
The ticket is #971